app/commands/git/sync_exercise_approach_introduction.rb

```diff
@@ -0,0 +1,23 @@
+class Git::SyncExerciseApproachIntroduction
+  include Mandate
+
+  initialize_with :exercise, :config
+
+  def call
+    exercise.update!(
```

Comment:
Is this update actually needed? It feels like the `Create` below is actually doing the work of creating. What extra does this do?